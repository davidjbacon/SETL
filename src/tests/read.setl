READ(a,b,c);
PRINT(a,b,c);
