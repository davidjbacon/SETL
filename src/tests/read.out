string_datum 666 {[[{[]}]]}
